### PR TITLE
Only show groups filter if current clinician has > 1 group

### PR DESCRIPTION
**Pull request by luketlancaster**
- [x] Add the Clubhouse Story ID: [ch21377]

### CODE REVIEW

File: src/js/apps/patients/view/view_app.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ export default App.extend({
     const groupId = this.getState('groupId') || groups.at(0).id;
     this.setState({ groupId });
 
+    if (this.groups.length === 1) return;
```
Comment (paulfalgout) [inline src/js/apps/patients/view/view_app.js:47]: ditto above

File: test/integration/patients/view.js [hunk 2]
```diff
@@ -250,7 +250,9 @@ context('view page', function() {
     cy
       .get('.list-page__filters')
       .contains('Another Group');
+  });
 
+  specify('clinician in only one group', function() {
```
Comment (paulfalgout) [inline test/integration/patients/view.js]: this looks like it should be specify `group filtering - new actions` or something like that.  and there needs to be _another_ specify with one group down above 282.

File: src/js/apps/patients/list/patients-all_app.js [hunk 3]
```diff
@@ -25,6 +25,8 @@ export default App.extend({
     const groupId = this.getState('groupId') || groups.at(0).id;
     this.setState({ groupId });
 
+    if (groups.length === 1) return;
```
Comment (paulfalgout) [inline src/js/apps/patients/list/patients-all_app.js:28]: can we add a bail like this for `0` above 25 as a cheap work around no groups?  I'm not sure what the endpoint does when no group filter is sent, but I imagine it'll either be 0 results or something that just leaves the spinner vs a console error.

Comment (paulfalgout) [reply src/js/apps/patients/list/patients-all_app.js:28]: actually, let's try it and if it's no results, this is probably worth doing.. but if it's a spinner, no groups should just be handled differently.

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/apps/patients/list/patients-all_app.js (hunk 3 above)
Comment (luketlancaster) [reply src/js/apps/patients/list/patients-all_app.js:28]: the endpoint 400's, so we'll probably need to handle it differently right?

Comment (paulfalgout) [reply src/js/apps/patients/list/patients-all_app.js:28]: yep ok

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**